### PR TITLE
KRACOEUS-8669: Display parent's status in child

### PR DESCRIPTION
**Pull request by gathreya**

### CODE REVIEW

File: coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentViewHelperServiceImpl.java [hunk 1]
```diff
@@ -725,6 +725,15 @@ protected boolean renderEditForPersonnelAttachment(String personId, DevelopmentP
         return true;
     }
 
+    public String getProposalParentStatus(DevelopmentProposal proposal) {
+        if (proposal.isChild()) {
+            ProposalDevelopmentDocument parentDoc = getProposalHierarchyService().getParentDocument(proposal.getProposalDocument());
```
Comment (leoherbie) [inline coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentViewHelperServiceImpl.java]: This seems a bit wasteful to get the entire Document just to get the status' description.  I wonder what we can do.

I'd rather see:  

Maybe a custom JPQL DAO?

Comment (gathreya) [inline coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentViewHelperServiceImpl.java]: Yes, I think I can add something to the DAO

**Comment by gathreya** [comment]
test this please.

**Comment by gathreya** [comment]
changed, just used an existing DAO method.

**Comment by leoherbie** [comment]
after talking with @gathreya there will be other changes needed.

**Comment by gathreya** [comment]
made the required changes

**Pull request merged**